<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>more_info</class>
 <widget class="QWidget" name="more_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marathon Skills 2017</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(79, 79, 79);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: white;</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white
</string>
        </property>
        <property name="text">
         <string>      MARATHON SKILLS 2017</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>400</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>BMR калькулятор</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>Список
благотворительных
организаций</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_8">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>BMI калькулятор</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>Marathon Skills 2017</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>Предыдущие
результаты</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>95</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(209, 209, 209);
border-style: outset;
border-width: 1px;
border-radius: 5px;
border-color: beige;
min-width: 7em;
padding: 1px;
border-color: grey;
color: black;
</string>
           </property>
           <property name="text">
            <string>Насколько долгий
марафон</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="lower_pamel" native="true">
     <property name="minimumSize">
      <size>
       <width>960</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(79, 79, 79)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
